internal/pkg/cli/app_delete.go

```diff
@@ -29,11 +29,14 @@ var (
 	errAppDeleteCancelled = errors.New("app delete cancelled - no changes made")
 )
 
-type deleteAppOpts struct {
-	// Flags or arguments that are user inputs.
+type deleteAppVars struct {
 	*GlobalOpts
 	SkipConfirmation bool
 	AppName          string
+}
+
+type deleteAppOpts struct {
+	*deleteAppVars
```

Comment:
should we make this not a pointer? so that when we embed the struct a copy is made instead of modifying the original object.